Tasks/NpmV1/task.json

```diff
@@ -3,7 +3,7 @@
     "name": "Npm",
     "friendlyName": "npm",
     "description": "Install and publish npm packages, or run an npm command. Supports npmjs.com and authenticated registries like Package Management.",
-    "helpUrl": "https://go.microsoft.com/fwlink/?LinkID=613746",
```

Comment:
Is the preferred style no-fwlinks?  It sure makes PR review easier, but it means any doc changes require a release...which means trouble for onprem potentially.

Reply:
Good question.  There are so many tasks that creating an fwlink for each that is opaque seemed difficult to manage.  The doc URLs are treated as stable - they are never changed without also adding a redirect to the changed content.